pkgs/development/python-modules/moto/default.nix

```diff
@@ -28,30 +27,31 @@
 
 buildPythonPackage rec {
   pname = "moto";
-  version = "1.3.14";
+  version = "1.3.16";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0fm09074qic24h8rw9a0paklygyb7xd0ch4890y4v8lj2pnsxbkr";
+    sha256 = "0zy0prsyip264i6h03lxsn1qg1n3dc8c4iyfawckjqvm24gnns3c";
   };
 
   postPatch = ''
     substituteInPlace setup.py \
-      --replace "jsondiff==1.1.2" "jsondiff~=1.1"
-    sed -i '/datetime/d' setup.py # should be taken care of by std library
+      --replace "ecdsa<0.15" "ecdsa"
   '';
 
   patches = [
-    # loosen idna upper limit
     (fetchpatch {
-      url = "https://github.com/spulec/moto/commit/649b497f71cce95a6474a3ff6f3c9c3339efb68f.patch";
-      sha256 = "03qdybzlskgbdadmlcg6ayxfp821b5iaa8q2542cwkcq7msqbbqc";
+      # Remove dependence on boto. The boto library  (long ago superseded by boto3)
+      # has not had an official release in over two years or even a commit in the
+      # last 18 months.
+      url = "https://github.com/spulec/moto/pull/3503.patch";
```

Comment:
fetch https://github.com/spulec/moto/commit/ae85c539fd57034c4d5cfd0f95af41ff19862dd1.patch instead of a pull patch